ray-operator/controllers/ray/batchscheduler/volcano/volcano_scheduler_test.go

```diff
@@ -79,7 +80,7 @@ func TestCreatePodGroup(t *testing.T) {
 
 	minMember := utils.CalculateDesiredReplicas(&cluster) + *cluster.Spec.HeadGroupSpec.Replicas
 	totalResource := utils.CalculateDesiredResources(&cluster)
-	pg := createPodGroup(&cluster, cluster.ClusterName, minMember, totalResource)
+	pg := createPodGroup(&cluster, fmt.Sprintf("ray-%s-pg", cluster.Name), minMember, totalResource)
```

Comment:
Maybe we can use `getAppPodGroupName` instead of hard-coding it?


https://github.com/ray-project/kuberay/blob/7f986b67cc68d691d426d4aefc46dd09fe3628f0/ray-operator/controllers/ray/batchscheduler/volcano/volcano_scheduler.go#L68

Reply:
Done.